src/script/calling/CallingRepository.ts

```diff
@@ -1992,6 +1992,10 @@ export class CallingRepository {
       .forEach(participant => participant.videoState(state));
   };
 
+  private readonly metricsReceived = (_: string, metrics_json: string) => {
+    this.logger.info('Calling metrics:', metrics_json);
```

Comment:
We using camelCase for function args :)

```suggestion
  private readonly metricsReceived = (_: string, metricsJSON: string) => {
    this.logger.info('Calling metrics:', metricsJSON);
```